ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/getridof/AbstractStep.java

```diff
@@ -1,4 +1,4 @@
-package io.syndesis.qe.pages.integrations.edit.steps;
+package io.syndesis.qe.pages.integrations.editor.add.steps.getridof;
```

Comment:
Might be sufficient to just annotate with `@Deprecated`

Reply:
those classes are "out of the system". I really want to get rid of them in the next PR. If it's not possible I'll make them deprecated. https://github.com/syndesisio/syndesis-qe/issues/425

Reply:
ok